ci/JENKINS.md

```diff
@@ -0,0 +1,88 @@
+# Installing Spinnaker for GCP on Jenkins
+
+Installing Spinnaker for GCP can be automated with Jenkins. The Jenkins agent executing the job must be installed on a Unix-like operating system. 
+
+The following section assumes the user has an existing Jenkins server. If not, consider one of the [Jenkins solutions on Google Cloud](https://cloud.google.com/jenkins/).
+
+## Jenkins on GCP
+
+If your Jenkins server is running on GCP, follow [best practices](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#best_practices) for managing its service account. Your Jenkins server must have full access to all Google Cloud APIs to successfully install Spinnaker for GCP.
```

Comment:
Yeah the instance is given full access to APIs but the actual IAM permissions are restricted by the IAM roles on its service account. That practice is lifted from the [best practices doc](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#best_practices). There is another doc that gives the steps for how to give an instance full Cloud API access, do you think it would be helpful to [link to it](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes) as well?

Reply:
Yes, that sounds like a good idea.